src/sentry/static/sentry/app/utils/withTeamsForUser.tsx

```diff
@@ -62,6 +72,18 @@ const withTeamsForUser = <P extends InjectedTeamsProps>(
       }
     }
 
+    populateTeamsWithProjects(teams: Team[], projects: Project[], isSuperuser: boolean) {
```

Comment:
@davidenwang You might consider omitting the `isSuperuser` input and just reading it from `ConfigStore`. Just a thought.

Reply:
mm yea that makes more sense